### PR TITLE
2015 08 mn add alex proposal detail view

### PR DESCRIPTION
**Pull request by MagdaN**
add proposal detail to alexanderplatz

### CODE REVIEW

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <div class="moving-column-tab">
+            <i class="icon-document moving-column-icon"></i>
+        </div>
+    </a>
+
+    <adh-recompile-on-change
+        data-ng-switch-when="body"
+        data-value="{{proposalUrl}}">
+        <adh-mein-berlin-proposal-detail
+            data-ng-show="proposalUrl"
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: Why did you put in this? I would prefer to either leave it out or use `ng-if`.

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/Workbench.ts [hunk 2]
```diff
@@ -95,6 +95,19 @@ export var documentDetailColumnDirective = (
...
 
+export var proposalDetailColumnDirective = (
+    adhConfig : AdhConfig.IService
+) => {
+    return {
+        restrict: "E",
+        templateUrl: adhConfig.pkg_path + pkgLocation + "/ProposalDetailColumn.html",
+        require: "^adhMovingColumn",
+        link: (scope, element, attrs, column : AdhMovingColumns.MovingColumnController) => {
+            column.bindVariablesAndClear(scope, ["processUrl", "proposalUrl"]);
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/Workbench.ts]: `proposalItemOptions` needs to be defined.

File: src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html [hunk 3]
```diff
@@ -1,14 +1,16 @@
 <article class="mein-berlin-proposal-detail">
     <header class="mein-berlin-proposal-detail-header">
+
         <span
-                data-ng-if="data.assignments.length > 0"
+                data-ng-if="data.assignments.length > 0 && isKiezkasse"
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html]: are badges specific to kiezkassen?

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<div data-ng-switch="transclusionId">
+    <div data-ng-switch-when="menu">
+        <div class="moving-column-tab">
+            <i class="icon-document moving-column-icon"></i>
+            {{ "TR__PROPOSAL" | translate }}
+        </div>
+        <a class="moving-column-menu-button" data-ng-if="proposalItemOptions.POST" data-ng-href="{{ proposalUrl | adhParentPath | adhResourceUrl:'edit' }}">{{ "TR__EDIT" | translate }}</a>
+
+        <div class="moving-column-menu-nav">
+            <a class="moving-column-menu-nav-button" data-ng-href="{{ processUrl | adhResourceUrl }}" title="{{ 'TR__CLOSE' | translate }}"><i class="icon-x moving-column-menu-nav-icon"></i></a>
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: needs to link to the 'proposals' view

File: src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html (hunk 3 above)
Comment (MagdaN) [inline src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html]: no, but the badge classes are that are defined in ng-class

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html (hunk 1 above)
Comment (MagdaN) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: This is the code from kiezkassen

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<div data-ng-switch="transclusionId">
+    <div data-ng-switch-when="menu">
+        <div class="moving-column-tab">
+            <i class="icon-document moving-column-icon"></i>
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: wrong icon

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        <a class="moving-column-menu-button" data-ng-if="proposalItemOptions.POST" data-ng-href="{{ proposalUrl | adhParentPath | adhResourceUrl:'edit' }}">{{ "TR__EDIT" | translate }}</a>
+
+        <div class="moving-column-menu-nav">
+            <a class="moving-column-menu-nav-button" data-ng-href="{{ processUrl | adhResourceUrl }}" title="{{ 'TR__CLOSE' | translate }}"><i class="icon-x moving-column-menu-nav-icon"></i></a>
+        </div>
+    </div>
+
+    <a class="moving-column-expand" data-ng-switch-when="collapsed" data-ng-href="{{ proposalUrl | adhParentPath | adhResourceUrl }}">
+        <div class="moving-column-tab">
+            <i class="icon-document moving-column-icon"></i>
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: wrong icon

File: src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html (hunk 3 above)
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html]: But still: If there are badges, they should be displayed. If the badge names are not used outside of kiezkassen they do no harm.

File: src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html (hunk 5 above)
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/MeinBerlin/Alexanderplatz/Workbench/ProposalDetailColumn.html]: Now that I look closer into this, it seems like the designs do not even have icons here. Strange.

**Pull request merged**